apple/internal/codesigning_support.bzl

```diff
@@ -450,7 +450,9 @@ def _codesigning_command(
         features = platform_prerequisites.features,
         rule_descriptor = rule_descriptor,
     )
-    if platform_prerequisites.platform.is_device or should_sign_sim_bundles:
+    if not should_sign_sim_bundles:
+        return ""
```

Comment:
this same condition is on 415, not sure if we are trying to rely on some of the side effects in between?

Reply:
415 is `should_sign_bundles` but this is checking `should_sign_sim_bundles`. But yes, it doesn't need any of the code between, so I should move `should_sign_sim_bundles` calculation up and make it an `or` there.